packages/cloud_firestore/cloud_firestore/example/integration_test/collection_reference_e2e.dart

```diff
@@ -169,6 +169,32 @@ void runCollectionReferenceTests() {
         },
         timeout: const Timeout.factor(3),
       );
+
+      testWidgets(
+        '#8082 - returning null from `fromFirestore` should not throw a null check error',
```

Comment:
We are not putting issues number in the test description (we already have this information with git history)

Reply:
Done